### DISCUSSION TITLE
App script - Install Button not available

### DISCUSSION THREAD

**Discussion by roberobs** [derekantrican/GAS-ICS-Sync#360] [Q&A]
Hi, 
I do not see the install button next to Debug button on Apps Scripts page. Also when clicking Install nothing seems to happen...When I run script once i got message it is completed. Just waiting to see the changes now in Google Cal.
Could you please advise how can I install the script?
Thanks!

**Comment by Lonestarjeepin** [comment]
Not sure if you were able to figure this out, but first you have to be in Code.gs.  Then, along the top you will see Run, Debug, and a dropdown where you can select Install.  Once you have Install selected, you will click the Run button.  In the Execution Log, you will only see one line of log text about "Intended frequency...".  That should be it.
